ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/rtb/VungleRtbBannerAd.java

```diff
@@ -82,38 +97,158 @@ public void render() {
       return;
     }
 
-    AdapterParametersParser.Config config = AdapterParametersParser.parse(appID, mediationExtras);
-    // Vungle SDK doesn't support rendering multiple banner ads for the same placement at the same time,
```

Comment:
@HuFangshuai Is this PR still needed for review? I remember that Android doesn't support it without large SDK change?